### ISSUE TITLE
Can't Compile gcc error

### ISSUE THREAD

**Issue by NickTheOnlyNeko** [aircrack-ng/rtl8188eus#206]
so i installed linux-lts and linux-lts-headers then I downgraded them to 5.10.90 and when i do `sudo make && make install` i get the following
> `make ARCH=x86_64 CROSS_COMPILE= -C /lib/modules/5.10.90-1-lts/build M=/home/nick/rtl8188eus  modules`
> `make[1]: Entering directory '/usr/lib/modules/5.10.90-1-lts/build'`
> `  CC [M]  /home/nick/rtl8188eus/core/rtw_cmd.o`
> `cc1: error: incompatible gcc/plugin versions`
> `cc1: error: failed to initialize plugin ./scripts/gcc-plugins/structleak_plugin.so`
> `make[2]: *** [scripts/Makefile.build:280: /home/nick/rtl8188eus/core/rtw_cmd.o] Error 1`
> `make[1]: *** [Makefile:1822: /home/nick/rtl8188eus] Error 2`
> `make[1]: Leaving directory '/usr/lib/modules/5.10.90-1-lts/build'`
> `make: *** [Makefile:2058: modules] Error 2`

OS Info:

![image](https://user-images.githubusercontent.com/76766408/190891309-31fad91a-eba9-4faf-b08d-bf0da6d0ab00.png)

**Comment by komradehieu** [comment]
Yeah i got the exact error like yours. I'm on 5.10.89-lts kernel. Hoping that this problem can be solved soon. 
![image](https://user-images.githubusercontent.com/69588235/202852125-c5e8d6f3-6d08-4d64-b35f-e01a869292f4.png)

**Comment by Berej** [comment]
me too

**Comment by Berej** [comment]
I solved. Need to install linux-headers

**Comment by komradehieu** [comment]
> I solved. Need to install linux-headers

Bro how does this happen ? I have already installed Linux-headers. Screenshot down below
![image](https://user-images.githubusercontent.com/69588235/207229801-22140a49-1c79-4401-848f-dcf494baa0d0.png)

**Comment by komradehieu** [comment]
I even have both linux-headers and linux -lts-headers. Still didnt work

![image](https://user-images.githubusercontent.com/69588235/207229959-064bd7ad-705f-4084-a9e6-7833f41afc15.png)

**Comment by NickTheOnlyNeko** [comment]
try installing these 4
`yay -Syu dkms bc linux-firmware linux-lts-headers`

if you don't have yay installed do pacman instead of yay or install yay / paru

**Comment by NickTheOnlyNeko** [comment]
> > I solved. Need to install linux-headers
> 
> Bro how does this happen ? I have already installed Linux-headers. Screenshot down below ![image](https://user-images.githubusercontent.com/69588235/207229801-22140a49-1c79-4401-848f-dcf494baa0d0.png)

btw what DE is that?

**Comment by komradehieu** [comment]
> > > I solved. Need to install linux-headers
> > 
> > 
> > Bro how does this happen ? I have already installed Linux-headers. Screenshot down below ![image](https://user-images.githubusercontent.com/69588235/207229801-22140a49-1c79-4401-848f-dcf494baa0d0.png)
> 
> btw what DE is that?

no actually im using no DE. This is just a tiling window manager. And my case im using BSPWM tiling window manager

**Comment by NickTheOnlyNeko** [comment]
> > > > I solved. Need to install linux-headers
> > > 
> > > 
> > > Bro how does this happen ? I have already installed Linux-headers. Screenshot down below ![image](https://user-images.githubusercontent.com/69588235/207229801-22140a49-1c79-4401-848f-dcf494baa0d0.png)
> > 
> > 
> > btw what DE is that?
> 
> no actually im using no DE. This is just a tiling window manager. And my case im using BSPWM tiling window manager

use latest stable Linux version (for me it's 6.1.4) and retry

would recommend this one to use [this one](https://github.com/KanuX-14/rtl8188eus)

**Comment by komradehieu** [comment]
> > > > > I solved. Need to install linux-headers
> > > > 
> > > > 
> > > > Bro how does this happen ? I have already installed Linux-headers. Screenshot down below ![image](https://user-images.githubusercontent.com/69588235/207229801-22140a49-1c79-4401-848f-dcf494baa0d0.png)
> > > 
> > > 
> > > btw what DE is that?
> > 
> > no actually im using no DE. This is just a tiling window manager. And my case im using BSPWM tiling window manager
> 
> use latest stable Linux version (for me it's 6.1.4) and retry
> 
> would recommend this one to use [this one](https://github.com/KanuX-14/rtl8188eus)
> 

update: im using latest lts kernel version and its working fine also. Monitor mode, aireplay-ng,... everything works, but sometimes its a little bit buggy. I tried zen kernel on my machine and ended up in booting directly to uefi instead of grub, have to chroot into arch again to delete zen kernel  and reinstall grub
